.github/workflows/latest_dependency_checker.yml

```diff
@@ -0,0 +1,36 @@
+name: Latest Dependency Checker
+on:
+  schedule:
+    - cron: '0 * * * *'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8.x'
+    - name: Install pip and virtualenv
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install virtualenv
+    - name: Update latest core dependencies
+      run: |
+        python -m virtualenv venv_core
+        source venv_core/bin/activate
+        python -m pip install --upgrade pip
+        python -m pip install -r requirements.txt
+        python -m pip install -r test-requirements.txt
+        make checkdeps OUTPUT_FILEPATH=nlp_primitives/tests/requirement_files/latest_core_dependencies.txt
+        cat nlp_primitives/tests/requirement_files/latest_core_dependencies.txt
+    - name: Update latest core dependencies
```

Comment:
I think this should be update latests complete dependencies

Reply:
Done